distribution/pom.xml

```diff
@@ -33,10 +33,15 @@
         <karaf.version>2.4.1</karaf.version>
         <pax.web.version>3.1.2</pax.web.version>
         <pax.cdi.version>0.8.0</pax.cdi.version>
+        <ddf.platform.app.version>2.8.0-SNAPSHOT</ddf.platform.app.version>
```

Comment:
This makes sense to have the properties here at the distribution/pom.xml level. However, we probably don't need them defined at distribution/ddf/pom.xml then. We have them defined in two places now.